pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2475,6 +2475,12 @@
     )
     private int transactionBufferClientMaxConcurrentRequests = 1000;
 
+    @FieldContext(
+            category = CATEGORY_TRANSACTION,
+            doc = "The transaction buffer client's operation timeout in milliseconds."
+    )
+    private long transactionBufferClientOperationTimeoutInMills = 3000;
```

Comment:
It's better to write in this way: `long transactionBufferClientOperationTimeoutInMills = 3000L;` or `int transactionBufferClientOperationTimeoutInMills = 3000;`

Reply:
thanks, fixed.